model_downloader/downloader.py

```diff
@@ -93,38 +228,31 @@ def get_confirm_token(response):
             return value
     return None
 
-def save_response_content(response, path, name, size):
-    destination = os.path.join(path, name)
-    chunk_size = 32768
-    with open(destination, 'wb') as f:
-        process_download(chunk_size, response, size, f)
-    print(name, '====>', destination)
-
 def delete_param(model):
-    tmpfile = os.path.join(args.output_dir, 'tmp.txt')
-    with open(model, 'r') as input_file, open(tmpfile, 'w') as output_file:
+    tmpfile = args.output_dir / 'tmp.txt'
```

Comment:
will it work under Windows?

Reply:
Yes, `pathlib` correctly handles path concatenation on both Windows & Linux.